src/main/java/org/semux/core/state/Account.java

```diff
@@ -60,7 +80,16 @@ public static Account fromBytes(byte[] address, byte[] bytes) {
         Amount locked = dec.readAmount();
         long nonce = dec.readLong();
 
-        return new Account(address, available, locked, nonce);
+        byte[] code = null;
+        Map<ByteArray, byte[]> storage = null;
+        try {
+            code = dec.readBytes();
+            storage = dec.readByteMap();
```

Comment:
Not good! Too much serialization and deserialization for accounts with big state. We can keep it for now; and create another ticket to address this issue.

Reply:
Yes, there is still much to do. :D